std/net/curl.d

```diff
@@ -1510,12 +1511,8 @@ private mixin template WorkerThreadProtocol(Unit, alias units)
     }
 }
 
-// Workaround bug #2458
```

Comment:
> as far as i understood 15831 it just increases the binary size

It's not just that. Your link times also increase exponentially with the symbol sizes. Also, debugging with symbols that are > 2000 characters sucks.

> and people are already working against this problem

People are working on compression of symbol names, which is a bandaid and not solution. You still end up with slower compile times, exponential symbol size, and an extra step to debugging.

> i think it's okay to set it private or move in the function.

Sure, set it to private, I'd argue that the above mentioned comment should be there as well.